FogCreek/Glitch-Community
# Oval cuckoo: filter My Stuff collections out of search results

## Links
* https://oval-cuckoo.glitch.me/
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/243

## Changes:
* filters out collections from search where `isMyStuff` is set to true. While "My Stuff" collections are not private, everyone will have a My Stuff collection so they won't really be particularly meaningful to search for. 
* we do however want to keep "My Stuff" searchable within the add project to collection popover

## How To Test:
* search for "My Stuff" on the search page, you'll see results but not as many as you should currently see on glitch.com and none of them should have the default description of "My place to save cool finds"
* go to /secret and enable my stuff, then create a my stuff collection by adding a project to the my stuff collection, then try adding another project to the collection by searching for the my stuff collection in the popover. "My Stuff" should show as an available option in the search results.
